packages/medusa/src/api/routes/admin/batch/index.ts

```diff
@@ -0,0 +1,41 @@
+import { Router } from "express"
+import { BatchJob } from "../../../.."
+import { DeleteResponse, PaginatedResponse } from "../../../../types/common"
+import middlewares from "../../../middlewares"
+
+const route = Router()
```

Comment:
As we have seen in other part of the code base, this should be in the exported function to avoid unexpected side effects with wrong usage